compiler/src/iree/compiler/Codegen/LLVMCPU/LLVMCPULowerExecutableTarget.cpp

```diff
@@ -177,17 +179,26 @@ void LLVMCPULowerExecutableTargetPass::runOnOperation() {
               moduleOp, translationInfo.value(),
               verifyConvTileAndDecomposeExpertConfig);
           break;
-        default:;
+        default:
+          break;
       }
       if (failed(verificationStatus)) {
         return signalPassFailure();
       }
 
+      auto maybeRootOp = getRootOperation(variantOp);
+      if (failed(maybeRootOp)) {
+        return signalPassFailure();
+      }
```

Comment:
It looks very wrong to me in this case, I think assertion is better. The assertion can stop the program immediately and raise the error message to users.